test/functional/p2p_timeouts.py

```diff
@@ -90,7 +90,7 @@ def run_test(self):
 
         with self.nodes[0].assert_debug_log(expected_msgs=expected_timeout_logs):
             self.mock_forward(3)
-            no_verack_node.wait_for_disconnect(timeout=1)
+            no_verack_node.wait_for_disconnect(timeout=2)
```

Comment:
I wonder why they specify the timeouts here and not just use the default value of 60